test/run_tests.jl

```diff
@@ -27,8 +27,8 @@ caching_optimizer.optimizer.inner.silent = true
 # See the docstring of MOI.Test.Config for other arguments.
 const CONFIG = MOI.Test.Config(
                                # Modify tolerances as necessary.
-                               atol = 5e-4,
-                               rtol = 5e-4,
+                               atol = 1e-3,
+                               rtol = 1e-3,

```

Comment:
Do you expect such loose tolerances on Mac and Windows?

Reply:
Looking into the problem, the found point super-optimal, but slightly infeasible. I tightened the tolerances on constraint violations, this should help.

Reply:
It seems that tightening the tolerances on the constraints has eliminated the issue.
There are two reasons why the tolerances are chosen quite loose:
1) we can't give any guarantees in terms of the distance of the optimal solution from the analytical one, since we terminate in terms of tolerances on the objective value.
2) for some of these test problems, lower/upper bounds are not given for all variables. But in the underlying method, we assume bounded domains. If no bounds are given by the user, [-1e8,1e8] is used. However, these large bounds can cause numerical inaccuracies in the subsolver, which will likely propergate to the returned solution.

Reply:
Looking into the problem, the found point super-optimal, but slightly infeasible. I tightened the tolerances on constraint violations and it seems that this has eliminated the issue.
There are two reasons why the tolerances are chosen quite loose:

   1. we can't give any guarantees in terms of the distance of the optimal solution from the analytical one, since we terminate in terms of tolerances on the objective value.
   2.  for some of these test problems, lower/upper bounds are not given for all variables. But in the underlying method, we assume bounded domains. If no bounds are given by the user, [-1e8,1e8] is used. However, these large bounds can cause numerical inaccuracies in the subsolver, which will likely propergate to the returned solution.